react/modules/nextjs-ssr-ssg/README.md

```diff
@@ -0,0 +1,41 @@
+# [React](https://github.com/rolling-scopes-school/tasks/tree/master/react) Next.js. SSR and SSG ⚙️
+
+## Module Overview 📚
+
+This module introduces students to Server Side Rendering and Static Site Generation with Next.js framework. It aims to provide an in-depth understanding of server rendering and difference between client and server computations. The module covers a range of topics from understating the principles of server rendering to implementing it with Next.js, and React Server Components.
+
+In addition, this module covers the Remix framework, as an analogue of Next.js.
```

Comment:
`as an analogue of Next.js.`
I would suggest changing this line to:
`as an alternative approach of enabling SSG and SSR for React-based applications`.

Reply:
Accepted